go/tasks/plugins/array/awsbatch/definition/job_def_cache.go

```diff
@@ -27,8 +29,9 @@ type CacheKey interface {
 }
 
 type cacheKey struct {
-	role  string
-	image string
+	role      string
+	image     string
+	structObj *structpb.Struct
```

Comment:
What is the reason to store the structObj in the cacheKey? I don't think I see it being used anywhere.

Reply:
sorry, my bad. 
I just updated it, and we will use `role`, `image`, `jobDefinitionInput` as cache key

Reply:
ohh that is a good catch, it would have failed the cache lookup